### PR TITLE
hypershift: Run the same tests for both periodics and presubmits

### PR DESCRIPTION
**Pull request by ironcladlou**
This commit adjusts all the AWS presubmits to run the same tests as
the periodics.

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 1]
```diff
@@ -272,7 +208,7 @@ tests:
         namespace: test-credentials
       dependencies:
       - env: OCP_IMAGE_LATEST
-        name: release:hosted-ocp-latest
+        name: release:latest
```
Comment (ironcladlou) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:211]: Looks like we weren't using the CPO built in the PR for this test before...

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 2]
```diff
@@ -566,11 +489,9 @@ tests:
...
```
Comment (ironcladlou) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:492]: Looks like we weren't using the CPO built in the PR for this test before...

**Comment by sjenning** [comment]
/lgtm
/hold
on e2e-aws rehearse

**Pull request closed without merging**